CHANGELOG.rst

```diff
@@ -7,11 +7,15 @@ All notable changes to the Maintenance NApp will be documented in this file.
 ********************************
 Added
 =====
+- Added persistence to maintenance using mongodb.
+- Added tracking of maintenance window creation and updated times.
 
 Changed
 =======
-
 - Updated REST API endpoints to include version numbers. Current version is `\v1`.
+- Changed API to use descriptive strings for `status` of maintenance windows instead of integers.
```

Comment:
Great enhancements, @Ktmi. If you could also update the [openapi.yml spec](https://github.com/kytos-ng/maintenance/blob/master/openapi.yml) to reflect the changes accordingly or if you want to handle that on another PR and issue that's supposed to land on this `2022.3` version is also OK.